pkgs/development/python-modules/ploomber-extension/default.nix

```diff
@@ -10,14 +10,14 @@
   ploomber-core,
   pytestCheckHook,
   pytest-jupyter,
+  stdenv,
 }:
 
 buildPythonPackage rec {
   pname = "ploomber-extension";
   version = "0.1.1";
 
   pyproject = true;
-  disabled = pythonOlder "3.6";
```

Comment:
You probably want to drop `pythonOlder` from the argument list too.

Reply:
Done